api/cmd/portainer/main.go

```diff
@@ -17,9 +17,11 @@ import (
 	"github.com/portainer/portainer/api/git"
 	"github.com/portainer/portainer/api/http"
 	"github.com/portainer/portainer/api/http/client"
+	"github.com/portainer/portainer/api/http/proxy"
+	"github.com/portainer/portainer/api/http/proxy/factory/kubernetes"
 	"github.com/portainer/portainer/api/internal/snapshot"
 	"github.com/portainer/portainer/api/jwt"
-	"github.com/portainer/portainer/api/kubernetes"
+	k8s "github.com/portainer/portainer/api/kubernetes"
```

Comment:
I don't mean to be pedant. but since this is the main kuberenetes package, maybe leave it as it was and rename the other one k8sProxy? seems more intuitive for me